### PR TITLE
feat: reshuffle folders structure

### PR DESCRIPTION
**Pull request by burakukula**
Main changes:
 - moving test files next to Component itself
 - renaming `styles` files to follow `Component.styles.ts`
 - renaming foldes from `component-name` to `ComponentName`
 
 If the package contains more than one component (Button, Badge, Forms, Cards), I added Folders for each component (where there is a README for the component) + REAMDE.md, which contains a list of components from this package.

### CODE REVIEW

File: packages/components/badge/src/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-export { Badge } from './Badge';
-export type { BadgeProps } from './Badge';
-export { EntityStatusBadge } from './EntityStatusBadge';
-export type { EntityStatusBadgeProps } from './EntityStatusBadge';
+export { Badge } from './Badge/Badge';
+export type { BadgeProps } from './Badge/Badge';
```
Comment (gui-santos) [inline packages/components/badge/src/index.ts:2]: I thought these two would stay in the src folder, no?

Comment (burakukula) [reply packages/components/badge/src/index.ts:2]: but we have 2 separate README files for them. how do you think we should handle those cases?

File: packages/components/accordion/src/Accordion.test.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { axe } from '@/scripts/test/axeHelper';
 
-import { Accordion } from '../src';
+import { Accordion } from '.';
```
[the remark this answers is not in the document]
Comment (burakukula) [reply packages/components/accordion/src/Accordion.test.tsx:6]: hmm, we actually want to import from index I guess, to be able to use all the compound components.

File: packages/components/badge/src/index.ts (hunk 1 above)
Comment (gui-santos) [reply packages/components/badge/src/index.ts:2]: Ah I sent this messages before we talked this morning

they are not relevant anymore :)

File: packages/components/accordion/src/Accordion.test.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (gui-santos) [reply packages/components/accordion/src/Accordion.test.tsx:6]: uhmmm I thought this rule of importing from index was only for code outside of the package and importing it

But I guess it does not matter, it's just my personal preference to not have imports from "dots" hahahah

**Review by gui-santos — APPROVED** [review]
🏆 🏆 🏆 🏆 🏆 🏆 🏆 

thanks for your patience 🙏

**Review by suevalov — APPROVED** [review]

**Pull request merged**